### PR TITLE
Pin VC++ version to 14.26 

### PR DESCRIPTION
**Pull request by peterjc123**
VC++14.27 fails to compile mkl-dnn, see oneapi-src/oneDNN#812

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .circleci/scripts/windows_vs_14_26_install.ps1 [hunk 1]
```diff
@@ -0,0 +1,21 @@
+$VS_INSTALLER = "C:\Program Files (x86)\Microsoft Visual Studio\Installer\vs_installer.exe"
+$COLLECT_DOWNLOAD_LINK = "https://aka.ms/vscollect.exe"
+$VS_INSTALL_ARGS = @("modify", "--nocache","--quiet","--wait",
+                                                     "--installPath ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community""",
+                                                     "--add Microsoft.VisualStudio.Component.VC.14.26.x86.x64")
+
+$process = Start-Process "$VS_INSTALLER" -ArgumentList $VS_INSTALL_ARGS -NoNewWindow -Wait -PassThru
+Remove-Item -Path vs_installer.exe -Force
```
Comment (malfet) [inline .circleci/scripts/windows_vs_14_26_install.ps1:8]: This doesn't have to be removed, does it?
```suggestion
```

**Comment by peterjc123** [comment]
@malfet The fix is working for PyTorch Windows builds.

**Review by guyang3532 — APPROVED** [review]

**Comment by albanD** [comment]
I think this missed the binary job tests specified in [here](https://github.com/pytorch/pytorch/blob/master/.circleci/verbatim-sources/job-specs/binary-job-specs.yml#L280-L334) ? 
The CI is still failing for this one: https://ezyang.github.io/pytorch-ci-hud/build1/pytorch-master?mode=binary

**Comment by malfet** [comment]
@albanD because we need to apply similar fix to  https://github.com/pytorch/builder , this PR presumably should address both issues by switching to older windows images https://github.com/pytorch/pytorch/pull/43220

**Pull request closed without merging**